mart/configs/attack/classification_fgsm_linf.yaml

```diff
@@ -0,0 +1,11 @@
+defaults:
+  - adversary
+  - fgm
+  - linf
+  - composer: additive
+  - gradient_modifier: sign
+  - gain: cross_entropy
+  - objective: misclassification
+
+eps: ???
```

Comment:
Why not `lr: ${eps}`?

Reply:
The learning rate of FGSM is the epsilon, which is inferred in `fgm.yaml`. So we only expose the `eps` parameter.